modules/update-common-terms.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-updates.adoc
+
+:_content-type: REFERENCE
+[id="update-common-terms_{context}"]
+= Common terms
+
+Control plane:: The _control plane_, which is composed of control plane machines, manages the {product-title} cluster. The control plane machines manage workloads on the compute machines, which are also known as worker machines.
+
+Cluster Version Operator:: The _Cluster Version Operator_ (CVO) checks with the OSUS to retrieve valid updates and update paths based on your current component versions and information in the graph.
```

Comment:
CVO is also responsible for starting the update process for the cluster.

Reply:
and also manages the manifests in the release image.  Maybe something like:

> The _Cluster Version Operator_ (CVO) reconciles the cluster's core {product-title} components with the target release payload.  It also retrieves update recommendations from the OSUS, and updates between releases at the adminstrator's request.

or similar?

Reply:
How about something like "CVO is also responsible for maintaining the cluster baseline version and starting the update process when that version changes" or some such that works in the constant reconciliation angle as well.

Reply:
We can not go to the technical details for this.